src/App/Tracker/Controller/Hooks/Listeners/JoomlacmsPullsListener.php

```diff
@@ -442,6 +442,31 @@ protected function checkPullLabels($hookData, Github $github, Logger $logger, $p
 		// Get the files modified by the pull request
 		$files = $this->getChangedFilesByPullRequest($hookData, $github, $logger, $project);
 
+		// DEBUG
+		if (is_array($files))
+		{
+			$logger->message(
```

Comment:
And there is no `message` function defined in the PSR-3 interface or `Monolog\Logger` 😉